relay/workflow.py

```diff
@@ -1953,7 +1953,7 @@ def __update_access_token(refresh_token:str, client_id:str):
     return access_token
 
 
-def trigger_workflow(access_token:str, refresh_token:str, client_id:str, workflow_id:str, subscriber_id:str, user_id:str, action_args:dict=None):
+def trigger_workflow(access_token:str, refresh_token:str, client_id:str, workflow_id:str, subscriber_id:str, user_id:str, targets, action_args:dict=None):
```

Comment:
Could you provide typing in the declaration so it must be an array?

Reply:
And this may be a bit out-of-scope for your change, but we should probably have inline docs for these methods.

Reply:
Nevermind on the "inline docs" comment, I didn't scroll far enough ;-)

Reply:
Fixed - Let met know if I did that wrong, I wasn't sure of the standards in python for passing an array as a parameter